zerver/models/recipients.py

```diff
@@ -140,6 +140,8 @@ class DirectMessageGroup(models.Model):
     # Foreign key to the Recipient object for this DirectMessageGroup.
     recipient = models.ForeignKey(Recipient, null=True, on_delete=models.SET_NULL)
 
+    group_size = models.IntegerField(db_index=True)
```

Comment:
How do you expect this database index to be used? 

I don't really see a use for a global database index on this field; one on `(realm, group_size)` could maybe be useful, but we don't have a `realm` column on this table. So I think this index might not be constructive.